google/cloud/spanner/samples/samples.cc

```diff
@@ -1140,6 +1140,34 @@ void DmlPartitionedUpdate(google::cloud::spanner::Client client) {
   std::cout << "Update was successful [spanner_dml_partitioned_update]\n";
 }
 //! [END spanner_dml_partitioned_update]
+
+//! [START spanner_dml_batch_update]
+void DmlBatchUpdate(google::cloud::spanner::Client client) {
+  namespace spanner = google::cloud::spanner;
+
+  auto txn = spanner::MakeReadWriteTransaction();
+  std::vector<spanner::SqlStatement> statements = {
+      spanner::SqlStatement("INSERT INTO Albums"
+                            " (SingerId, AlbumId, AlbumTitle, MarketingBudget)"
+                            " VALUES (1, 3, 'Test Album Title', 10000)"),
+      spanner::SqlStatement("UPDATE Albums"
+                            " SET MarketingBudget = MarketingBudget * 2"
+                            " WHERE SingerId = 1 and AlbumId = 3")};
+  auto result = client.ExecuteBatchDml(txn, statements);
```

Comment:
Our documentation for this function says:

https://github.com/googleapis/google-cloud-cpp-spanner/blob/488c88db60b8e280ee1f4989d97c6aedbd38e386/google/cloud/spanner/client.h#L408-L410

So, I wonder if this sample should also use a Commit mutator functor?